### ISSUE TITLE
UNIFI conversion to mzML using DDA data

### ISSUE THREAD

**Issue by MHJena** [ProteoWizard/pwiz#1256]
Hey,

currently it seems like exporting mzML files from a UNIFI database is not supported for DDA data ("only MSe and HD-MSe data supported at this time).

Is there any other way to export a mzML containing MS1 AND MS2 information from UNIFI? Is there a systematic reason why DDA data is not supported?

Thank you very much!

**Comment by chambm** [comment]
Yes there's a reason: the UNIFI API doesn't support spectra access for anything except MSe.

**Comment by gzu300** [comment]
The unifi reference api shows under Sample results there are 2 endpoints:
'/sampleresults({resultID})/spectra/mass.dda'

Can these 2 endpoints be used to download dda data?
![screenCapture_18754390_143985245_0](https://user-images.githubusercontent.com/33227482/166451094-d9a35341-1a2d-4590-b491-46eb80a31d66.png)
